src/PackagerNewCommand.php

```diff
@@ -101,7 +101,12 @@ public function handle()
         // Creating a Laravel Service Provider in the src directory
         $this->info('Creating service provider...');
             $newProvider = $fullPath.'/src/'.$name.'ServiceProvider.php';
-            $this->helper->replaceAndSave(__DIR__.'/ServiceProvider.stub', ['{{vendor}}', '{{name}}'], [$vendor, $name], $newProvider);
+            $this->helper->replaceAndSave(
+                \Config::get('packager.service_provider_stub', __DIR__.'/ServiceProvider.stub'),
```

Comment:
Instead of providing a default here, place the default in the config file to make it more clear.

Reply:
I thought about this, but the problem is the location of the file, because a config is published in the config folder, you can't use __DIR__. And we cannot assume that this package is installed with composer and lives in the vendor folder. Thats why I came to this solution.

Reply:
Fair enough :)